services/app-api/prisma/schema.prisma

```diff
@@ -306,6 +306,16 @@ model State {
   rates                       RateTable[]
```

Comment:
I wonder if we should add the relationship between `StateProgram` and `State`.

Reply:
I wondered this too. Are you thinking `StateProgram` has a `State` and a `State` has a list of `StatePrograms`?

Reply:
Like this

```postgres
model State {
  stateCode                   String                   @id @default(uuid())
  name                        String
  latestStateSubmissionNumber Int                      @default(0)
  latestStateRateCertNumber   Int                      @default(0)
  statePackages               HealthPlanPackageTable[]
  users                       User[]
  contracts                   ContractTable[]
  rates                       RateTable[]
  statePrograms               StateProgram[]
}

model StateProgram {
  id                 String                @id @default(uuid())
  createdAt          DateTime              @default(now())
  updatedAt          DateTime              @default(now()) @updatedAt
  fullName           String
  name               String
  stateCode          String
  isRateProgram      Boolean
  state              State   @relation(fields: [stateCode], references: [stateCode])
}
```

Reply:
Yes this is great, I'm trying for us to be able to use the stateCode as the ID for states since it's such a good one. No need to make UUIDs when we have human readable two character identifiers